scripts/build-readme.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+
+const getPackageJSON = require('./get-package-json')
+
+const packageJSON = getPackageJSON()
+const path = './README.md'
+
+function getDocsPath() {
+  switch (packageJSON.name) {
+    case 'jss':
+      return 'setup'
+    default:
+      return packageJSON.name
+  }
+}
+
+const contents = `
+# ${packageJSON.name}
+
+> ${packageJSON.description}
+
+See our website [${
+  packageJSON.name
+}](https://cssinjs.org/${getDocsPath()}?v=v10.0.0-alpha.8) for more information.
```

Comment:
hardcoded version)))

Reply:
can we keep them as a separate helper script outside of rollup config to keep the config more readable and declarative?